lib/endpoints/class-wp-rest-controller.php

```diff
@@ -250,7 +250,6 @@ public function get_collection_params() {
 				'default'            => 10,
 				'minimum'            => 1,
 				'maximum'            => 100,
-				'sanitize_callback'  => 'absint',
```

Comment:
I tried that on some sites like following.
https://firegoby.jp/wp-json/wp/v2/posts?per_page=-1